### PR TITLE
Feature/login page frontend

### PR DESCRIPTION
**Pull request by ryanwooa**
Login page and Signup Page

### CODE REVIEW

**Comment by tpan501** [comment]
image
works fine when the browser window size is below 100% but shows grey div when the size is >= 100%...
Maybe it's my browser problem

**Comment by tpan501** [comment]
image
do we need to have a same gap between these three elements?

**Comment by ryanwooa** [comment]
> image do we need to have a same gap between these three elements?

Edit

**Comment by ryanwooa** [comment]
> image works fine when the browser window size is below 100% but shows grey div when the size is >= 100%... Maybe it's my browser problem

>= 100%?
Now, the minimum size of page is 750 * 630, if the window size less than this number, the scrollbar will show.

**Comment by tpan501** [comment]
Yeah now the login page looks much normal to me

How to reply other's response in this guthub page, not just comment..

File: frontend/public/login_pic.png
Comment (zli213) [inline frontend/public/login_pic.png:1]: I remember you mentioned before about putting all photos under the assets folder, or did I remember it wrong? Also, I suggest creating a new folder named "image" to store all photos collectively.

**Review by zli213 — APPROVED** [review]

File: frontend/public/login_pic.png
Comment (ryanwooa) [reply frontend/public/login_pic.png:1]: When writing the code, I found out that it's better to put it in public. I will change it in next commit.

Then it's still possible to create an image folder in public to manage, and then create another folder for avatar

**Review by jlmjkfd — COMMENTED** [review]
Does it look better if I set the buttons and the input the same size?
<div style="display:flex">



</div>

**Comment by ryanwooa** [comment]
> Does it look better if I set the buttons and the input the same size?
> 
>  

Yes, the original one is too small. But the same width is a little bit wired? So I adjust it to a bit longer. Is it OK?
image

**Review by tpan501 — APPROVED** [review]

**Review by xinranwangxwan559 — APPROVED** [review]

**Comment by jlmjkfd** [comment]
> > Does it look better if I set the buttons and the input the same size?
> >  
> 
> Yes, the original one is too small. But the same width is a little bit wired? So I adjust it to a bit longer. Is it OK? image

If you search 'login page' on Google, most designs have inputs and buttons of the same length. Only a few of the lengths are different. I think the same length is better.

**Review by jlmjkfd — APPROVED** [review]

**Comment by ryanwooa** [comment]
> > > Does it look better if I set the buttons and the input the same size?
> > >  
> > 
> > 
> > Yes, the original one is too small. But the same width is a little bit wired? So I adjust it to a bit longer. Is it OK? image
> 
> If you search 'login page' on Google, most designs have inputs and buttons of the same length. Only a few of the lengths are different. I think the same length is better.

OK, I'll adjust it to a same length

**Pull request merged**